src/platform/site-wide/sass/consolidated-patches.scss

```diff
@@ -15,10 +15,14 @@
 }
 
 header.merger {
-  z-index: 999;
+  z-index: 300;
```

Comment:
How come the z-index had to change?

Reply:
@ncksllvn header is so high that it causes overlap issue with VCL... Footer and crisis modal are set to 301